src/main/java/io/openbpm/control/service/job/impl/JobServiceImpl.java

```diff
@@ -125,11 +138,26 @@ public String getErrorDetails(String jobId) {
 
     @Override
     public String getHistoryErrorDetails(String jobId) {
-        ResponseEntity<String> response = historyApiClient.getStacktraceHistoricJobLog(jobId);
-        if (response.getStatusCode().is2xxSuccessful()) {
-            return Strings.nullToEmpty(response.getBody());
+        try {
+            ResponseEntity<String> response = historyApiClient.getStacktraceHistoricJobLog(jobId);
```

Comment:
Check the client library. If an error is returned, it means there's a bug in the library. In this case, you need to implement your custom request directly.